src/garage/torch/policies/base.py

```diff
@@ -0,0 +1,53 @@
+
+
+class Policy:
```

Comment:
The rest of garage uses the `abc` module to define base classes, please do so here as well.
An example is [here](https://github.com/rlworkgroup/garage/blob/8b1bac869ac2bb167aa9982c80ee968e662b9f5a/src/garage/sampler/base.py).

Reply:
I am just wondering there is no abc.ABC in [Policy](https://github.com/rlworkgroup/garage/blob/8b1bac869ac2bb167aa9982c80ee968e662b9f5a/src/garage/tf/policies/base.py#L4) and [Policy2](https://github.com/rlworkgroup/garage/blob/8b1bac869ac2bb167aa9982c80ee968e662b9f5a/src/garage/tf/policies/base2.py#L7) class
Is there any difference with others?

Reply:
I think that the main reason is just that those two are rather old, before we started using `abc`.
One could also argue that's there's a lot of code in `Policy2`, and only a few methods to overwrite, so it's not *really* a base class.

I think we're moving away from using `garage.tf.policies.Policy`, since it uses `Parameterized`.

Reply:
thanks! I will put abc.ABC and "@abc.abstractclassmethod"